frontend/src/components/AdminKubeConfigRequest.vue

```diff
@@ -0,0 +1,66 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <action-button-dialog
+    :shoot-item="shootItem"
+    @dialog-opened="onConfigurationDialogOpened"
+    ref="actionDialog"
+    width="500"
+    caption="Configure Kubeconfig Lifetime">
+    <template v-slot:actionComponent>
+      <wildcard-select
+        v-model="expiration"
+        :wildcard-select-items="expirations"
+        wildcard-select-label="Lifetime"
+        ></wildcard-select>
+    </template>
+  </action-button-dialog>
+</template>
+
+<script>
+import ActionButtonDialog from '@/components/dialogs/ActionButtonDialog'
+import WildcardSelect from '@/components/WildcardSelect'
+import shootItem from '@/mixins/shootItem'
+
+export default {
+  name: 'admin-kubeconfig-request',
```

Comment:
```suggestion
```

Reply:
The component is not used anywhere in the frontend.

Reply:
Fixed :)